commandline/src/main/java/tech/pegasys/web3signer/commandline/subcommands/Eth2SubCommand.java

```diff
@@ -190,7 +181,7 @@ private void logNetworkSpecInformation() {
 
   private Eth2NetworkConfiguration createEth2NetworkConfig() {
     Eth2NetworkConfiguration.Builder builder = Eth2NetworkConfiguration.builder();
-    builder.applyNetworkDefaults(network);
+    builder.applyNetworkDefaults(network).kzgNoop(true);
```

Comment:
Would this just be ignored if the chain was pre-deneb?

Reply:
Yeah, it will be